homeassistant/core.py

```diff
@@ -1251,11 +1261,7 @@ def __init__(
                 "Format should be <domain>.<object_id>"
             )
 
-        if len(state) > MAX_LENGTH_STATE_STATE:
-            raise InvalidStateError(
-                f"Invalid state encountered for entity ID: {entity_id}. "
```

Comment:
We no longer have the entity id in the error now, should we add that back?

Reply:
No need, it's logged by the entity helper.